bridge/client/app/_views/ktb-uniform-view/ktb-uniform-view.component.html

```diff
@@ -1,7 +1,13 @@
-<div fxFlexFill>
-  <div fxLayout="column">
-    <div class="container" fxLayout="column" fxFlex="0 0 calc(100%-35px)">
-      <ktb-keptn-services-list class="overflow-y-auto" fxFlex="0 0 calc(100%-15px)" [uniformRegistrations]="(uniformRegistrations$ | async)" (selectedServiceChanged)="selectService($event)"></ktb-keptn-services-list>
+<div class="container" fxLayout="column" fxFlex="100%">
```

Comment:
Insert parent container `<div fxFlexFill>`, else it will look like this when scrolling.
![image](https://user-images.githubusercontent.com/11599148/122027278-47b48480-cdcb-11eb-8f2f-ea63993d43cd.png)

Reply:
Done